src/views/tutorial-view/components/variants-dropdown-disclosure/variants-dropdown-disclosure.module.css

```diff
@@ -0,0 +1,8 @@
+.root {
```

Comment:
These styles don't account for super long variant option names at the moment. I'm leaning towards making a fix for that if it seems like the real data will have option names that long. I could be persuaded otherwise though.